### PR TITLE
Migrate expression namespace to AdaptiveExpressions

### PR DESCRIPTION
**Pull request by Danieladu**
Migrate expression namespace from Microsoft.Bot.Expressions to AdaptiveExpressions,
ref: #3249

### CODE REVIEW

File: libraries/AdaptiveExpressions/AdaptiveExpressions.csproj [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <PackageId>Microsoft.Bot.Expressions</PackageId>
+    <PackageId>AdaptiveExpressions</PackageId>
```
Comment (tomlm) [inline libraries/AdaptiveExpressions/AdaptiveExpressions.csproj]: This needs to be Adaptive.Expressions with a period.  Adaptive already has a number of a packages which use the dot nomenclature
Adaptive.WPF
Adaptive.Xaml
etc.

File: libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/EmitEvent.cs [hunk 2]
```diff
@@ -5,7 +5,7 @@
 using System.Runtime.CompilerServices;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Bot.Expressions.Properties;
+using AdaptiveExpressions.Properties;
```
Comment (tomlm) [inline libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/EmitEvent.cs]: >AdaptiveExpressions [](start = 6, length = 19)

Likewise, Adaptive.Expressions

File: libraries/AdaptiveExpressions/AdaptiveExpressions.csproj (hunk 1 above)
Comment (tomlm) [reply libraries/AdaptiveExpressions/AdaptiveExpressions.csproj]: Hmmm..Actually, they are AdaptiveCards.WPF, AdaptiveCards.Xaml etc. 
Do we want Adaptive or AdaptiveCards?

---
In reply to: [377424557](https://github.com/microsoft/botbuilder-dotnet/pull/3375#discussion_r377424557) [](ancestors = 377424557)

Comment (Danieladu) [reply libraries/AdaptiveExpressions/AdaptiveExpressions.csproj]: @vishwacsena maybe we should confirm this?

File: libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/EmitEvent.cs (hunk 2 above)
Comment (vishwacsena) [reply libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/EmitEvent.cs]: Was reviewed in last adaptive cards working group meeting. AdaptiveExpressions was were we landed. @matthidinger.

File: libraries/AdaptiveExpressions/AdaptiveExpressions.csproj (hunk 1 above)
Comment (boydc2014) [reply libraries/AdaptiveExpressions/AdaptiveExpressions.csproj]: yes, let @vishwacsena  to confirm, i thought we should call this AdapativeExpression

File: libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/EmitEvent.cs (hunk 2 above)
Comment (boydc2014) [reply libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/EmitEvent.cs]: Haven't seen this, so we are landed on AdapativeExpresion, I added an approve on this

**Comment by fuselabs** [comment]
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.dll** compared against [version 4.6.3](https://www.nuget.org/packages/Microsoft.Bot.Builder/4.6.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.Luis.dll** compared against [version 4.6.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.AI.Luis/4.6.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.QnA.dll** compared against [version 4.6.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.AI.QnA/4.6.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.ApplicationInsights.dll** compared against [version 4.6.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.ApplicationInsights/4.6.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Azure.dll** compared against [version 4.6.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.Azure/4.6.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Dialogs.dll** compared against [version 4.6.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.Dialogs/4.6.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Integration.ApplicationInsights.Core.dll** compared against [version 4.6.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.Integration.ApplicationInsights.Core/4.6.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Integration.AspNet.Core.dll** compared against [version 4.6.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.Integration.AspNet.Core/4.6.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.TemplateManager.dll** compared against [version 4.6.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.TemplateManager/4.6.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Testing.dll** compared against [version 4.6.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.Testing/4.6.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Configuration.dll** compared against [version 4.6.3](https://www.nuget.org/packages/Microsoft.Bot.Configuration/4.6.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Connector.dll** compared against [version 4.6.3](https://www.nuget.org/packages/Microsoft.Bot.Connector/4.6.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Schema.dll** compared against [version 4.6.3](https://www.nuget.org/packages/Microsoft.Bot.Schema/4.6.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Streaming.dll** compared against [version 4.6.3](https://www.nuget.org/packages/Microsoft.Bot.Streaming/4.6.3)

**Pull request merged**